File: logic/v1/system_models/Hotel.py
from django.db import models
from .Geography import City

class HotelClass(models.Model):
    hotel_class =models.CharField(max_length=300)
    def __str__(self):
        return self.hotel_class
class Hotel(models.Model):
    hotel_name = models.CharField(max_length=300)
    hotel_class = models.ForeignKey(HotelClass,blank=True,on_delete=models.DO_NOTHING,null=True)
    address = models.CharField(max_length=300)
    capacity =models.DecimalField(decimal_places=0,max_digits=12)
    cost =models.DecimalField(decimal_places=0,max_digits=12)
    city = models.ForeignKey(City,blank=True,on_delete=models.DO_NOTHING,null=True)
    def __str__(self):
        return self.hotel_name    

File: cli/api/data/hotel.go
package data

type HotelRoom struct {
	ID         int    `json:"id"`
	HotelName  string `json:"hotel_name"`
	HotelClass string `json:"hotel_class"`
	Cost       int    `json:"cost"`
	City       string `json:"city"`
	Address    string `json:"address"`
}
